### PR TITLE
Set capability issue handler in capability value migration

### PR DESCRIPTION
**Pull request by SupunS**
Fixes https://flow-foundation.slack.com/archives/C074F78ECNL/p1723052086114979

### CODE REVIEW

File: cmd/util/ledger/migrations/cadence_values_migration.go [hunk 1]
```diff
@@ -384,20 +386,32 @@ func NewCadence1CapabilityValueMigration(
...
 			_ *interpreter.Interpreter,
-			_ environment.Accounts,
+			accounts environment.Accounts,
 			reporter *cadenceValueMigrationReporter,
 		) []migrations.ValueMigration {
+
+			idGenerator := environment.NewAccountLocalIDGenerator(
+				tracing.NewMockTracerSpan(),
+				util.NopMeter{},
+				accounts,
```
Comment (SupunS) [inline cmd/util/ledger/migrations/cadence_values_migration.go:403]: Not sure if thi `idGenerator` needs to be shared between this (capability) migration and the link value migration.

Comment (turbolent) [reply cmd/util/ledger/migrations/cadence_values_migration.go:403]: @onflow/flow-cadence-execution

File: cmd/util/ledger/migrations/cadence_values_migration_test.go [hunk 2]
```diff
@@ -2133,3 +2133,136 @@ func TestCadenceValueMigrationEntry_MarshalJSON(t *testing.T) {
...
+
+	require.Equal(t, reporter.entries, []any{
+		capabilityMigrationEntry{
+			AccountAddress: testAddress,
+			AddressPath: interpreter.AddressPath{
+				Address: testAddress,
+				Path:    interpreter.NewUnmeteredPathValue(common.PathDomainStorage, "foo"),
+			},
+			BorrowType:   borrowType,
+			CapabilityID: 6,
```
Comment (turbolent) [inline cmd/util/ledger/migrations/cadence_values_migration_test.go:2257]: How come the ID is 6 here? Is it because the account already has some other IDs?

**Review by turbolent — APPROVED** [review]
Thank you for fixing this by adding the new required handler. Apologies, I had forgotten to add it when I updated flow-go to the latest Cadence version

**Review by j1010001 — APPROVED** [review]

File: cmd/util/ledger/migrations/cadence_values_migration_test.go (hunk 2 above)
Comment (SupunS) [reply cmd/util/ledger/migrations/cadence_values_migration_test.go:2257]: yeah, some id's get generated during the [bootstrapping payloads](https://github.com/onflow/flow-go/pull/6303/files#diff-241fc91b1acd5ed674adbeddc2d22f9e6ccf034c607218605f934f1e01029e53R2152). Don't know exactly what those values are though.

**Pull request merged**